mdpow/tests/test_analysis.py

```diff
@@ -67,12 +64,8 @@ def fep_benzene_directory(tmpdir_factory):
 class TestAnalyze(object):
     def get_Gsolv(self, pth):
         gsolv = pth.join("FEP", "water", "Gsolv.fep")
-        if sys.version_info.major == 2:
-            G = pickle.load(gsolv.open())
-        elif sys.version_info.major == 3:
-            # Needed to read old pickle files
-            with open(gsolv, 'rb') as f:
-                G = pickle.load(f, encoding='latin1')
+        with open(gsolv, 'rb') as f:
```

Comment:
add comment to say that we do this because the test pickles were pickled with Python 2.

Perhaps it would be even better to write them now with Python 3.

Reply:
I'm sure how the originals were generated and as not sure if it would just break more things. When it writes a pickle can the current tools load if it was generated in python 3?

Reply:
I think we can leave them as Python 2, as long as our code can run them even under Python 3.